src/shared/components/palette/complex/ccd-cyapage-label-filter.pipe.ts

```diff
@@ -0,0 +1,28 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { CaseField } from '../../../domain';
+
+@Pipe({
+  name: 'ccdCYAPageLabelFilter'
+})
+export class CcdCYAPageLabelFilterPipe implements PipeTransform {
```

Comment:
_Just an observation:_
Interesting naming convention here. "CCD" is an acronym as is "CYA", yet they're treated very differently. Looks a bit weird.

Reply:
this is has been a long pet hate but unfortunately inherited from ccd to us and we have just ran with it.

Reply:
Don't you mean it was inherited from Ccd? 😈